### DISCUSSION TITLE
Gauge xray-report

### DISCUSSION THREAD

**Discussion by patricia2193** [getgauge/gauge#2052] [General]
Hi! 
I'm trying to use the gauge xray-report which was created by @jbadeau (https://github.com/jbadeau/xray-report) but it doesn't seem to work: it logs a json in the IDE console, but nothing is imported in Jira. Besides that, it gets constantly forcefully killed because it does not exist after 8s (error: Plugin [xray-report] with pid [28048] did not exit after 8.00 seconds. Forcefully killing it.)

I thought that it might be the proxy, but I'm not sure. Anyway, could someone please advise?

**Comment by jbadeau** [comment]
Please log your gauge version and plugin version.

Please add the log.

Are you using cloud or sever jira?
What auth type are you using?

**Reply by patricia2193** [reply]
Here's what I have after checking the versions: 

Gauge version: 1.2.1

Plugins
-------
html-report (4.0.12)
java (0.7.15)
screenshot (0.1.0)
xml-report (0.2.3)
xray-report (0.3.2)

Regarding Jira, I will have to find out because I'm not sure - but I believe it's server. 
The auth type is basic.

**Reply by jbadeau** [reply]
Can you please add the log

**Reply by patricia2193** [reply]
Since it's a work related project, I cannot share the log publicly. The log contains certain information that cannot be shared. Can you please tell me what I should be looking into in the log and also in which log - I assume you mean the gauge log ?

**Reply by patricia2193** [reply]
I just sent a POST request using the XRAY api for importing XML files: /rest/raven/1.0/import/execution/junit
